extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/LambdaHttpHandler.java

```diff
@@ -165,12 +164,15 @@ private boolean isBinary(String contentType) {
         if (contentType != null) {
             int index = contentType.indexOf(';');
             if (index >= 0) {
-                return LambdaContainerHandler.getContainerConfig().isBinaryContentType(contentType.substring(0, index));
+                return isBinaryContentType(contentType.substring(0, index));
             } else {
-                return LambdaContainerHandler.getContainerConfig().isBinaryContentType(contentType);
+                return isBinaryContentType(contentType);
             }
         }
         return false;
     }
 
+    private boolean isBinaryContentType(String contentType) {
+        return contentType != null && !contentType.startsWith("text/");
```

Comment:
Should that test against `application/` types too?

Reply:
Please provide an option to read Content-Types from quarkus application properties so that application developers can decide.